src/chronometer.js

```diff
@@ -21,10 +34,13 @@ class Chronometer {
 
   stop() {
     // ... your code goes here
+    clearInterval(this.currentTime)
```

Comment:
Aquí estas deteniendo tu contador. 
Lo correcto sería guardar el Id de tu intervalo en `this.intervalId` y cancelarlo en lugar de `this.currentTime`.